_posts/en/newsletters/2023-04-05-newsletter.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Bitcoin Optech Newsletter #245'
+permalink: /en/newsletters/2023/04/05/
+name: 2023-04-05-newsletter
+slug: 2023-04-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Watchtower accountability proofs:** Sergi Delgado Segura
+  [posted][segura watchtowers post] to the Lightning-Dev mailing list
+  last week about holding [watchtowers][topic watchtowers] accountable
+  for failing to respond to protocol breaches they were capable of
+  detecting.  For example, Alice provides a watchtower with data for
+  detecting and responding to the confirmation of an old LN channel
+  state; later, that state is confirmed but the watchtower fails to
+  respond.  Alice would like to be able to hold the watchtower operator
+  accountable by publicly proving it failed to respond appropriately.
+
+    The basic principle would be for a watchtower to have a well-known
+    public key and to use the corresponding private key to generate a
+    signature for any breach-detection data it accepted.  Alice could
+    then publish the data and the signature after an unresolved breach
+    to prove the watchtower failed its responsibility.  However, Segura
```

Comment:
Sergi’s first family name is “Delgado”, “Segura” is his second family name. He also [refers to himself](https://twitter.com/sr_gi) as “Sergi Delgado” on Twitter.

```suggestion
    to prove the watchtower failed its responsibility.  However, Delgado
```

Reply:
It means safe (in feminine), but it is, indeed, also my second last name.

I usually go by Delgado because it makes things easier (which, while we are at it, means thin in Spanish lol)